docs/painless/painless-api-reference/painless-api-reference-interval/index.asciidoc

```diff
@@ -0,0 +1,9 @@
+[[painless-api-reference-interval]]
```

Comment:
I think we should have a note in any generated files. I havne't checked if this is generated though. If it isn't generated, I think it probably should be.

Reply:
It's generated.  Will add a note.

Reply:
:+1: